SPECS/python-pam/python-pam.spec

```diff
@@ -1,23 +1,28 @@
 Summary:        Python PAM module using ctypes
 Name:           python-pam
-Version:        1.8.4
-Release:        3%{?dist}
+# Update the version field in fix-pyproject.patch whenever this changes.
+Version:        2.0.2
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Azure Linux
 Group:          Development/Languages/Python
 URL:            https://pypi.python.org/pypi/python-pam/
 Source0:        https://pypi.python.org/packages/source/p/python-pam/%{name}-%{version}.tar.gz
```

Comment:
This would need to be updated to match the cgmanifest url

Reply:
Done